README.md

```diff
@@ -281,8 +281,10 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 
 *Libraries for network programming.*
 
+* [Grizzly](https://grizzly.java.net/) - NIO framework. Used as a network layer in Glassfish.
 * [Netty](http://netty.io/) - A framework for building high performance network applications.
 * [OkHttp](http://square.github.io/okhttp/) - An HTTP+SPDY client for Android and Java applications.
+* [Undertow](http://undertow.io/) - Web server providing both blocking and non-blocking API’s based on NIO. Used as a network layer in WildFly.
```

Comment:
A valid use case for Grizzly would be the same as for Netty, they actually have very similar purpose. Undertow is made for custom HTTP servers.